database/migrations/2020_05_04_150000_create_bookings_table.php

```diff
@@ -23,7 +23,10 @@ public function up()
             $table->morphs('experience');
             $table->morphs('subject');
             $table->foreignIdFor(app('rate'));
+            // The Escape Room Rate has a relationship with Booking, but there is no escaperoom_rate_id. I added this per https://tgerdev.slack.com/archives/C01LEEBRH33/p1620181624188200
+            $table->foreignIdFor(app('escaperoom_rate'));
```

Comment:
I don't think we want escape_room rate since we want Bookings to be generic. We need to either attach EscapeRoomRate to the rate column or remove it.